### PR TITLE
express example for node-sdk

### PR DESCRIPTION
**Pull request by roncohen**
Express.js example to show how to use the Node.js SDK

### CODE REVIEW

File: packages/node-sdk/example/bucket.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { BucketClient } from "../src";
+
+export const bucket = new BucketClient({
+  secretKey: process.env.BUCKET_SECRET_KEY || "",
+});
+
+await bucket.initialize();
```
Comment (pavkam) [inline packages/node-sdk/example/bucket.ts:7]: Does this `await` actually work at module level?

File: packages/node-sdk/example/bucket.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { BucketClient } from "../src";
+
+export const bucket = new BucketClient({
+  secretKey: process.env.BUCKET_SECRET_KEY || "",
```
Comment (pavkam) [inline packages/node-sdk/example/bucket.ts:4]: We need to settle on the fallback flags and maybe add some here for reference?

File: packages/node-sdk/example/app.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+const todos = ["Buy milk", "Walk the dog"];
+
+app.get("/", (req, res) => {
+  res.locals.bucketUser.track("Front Page Viewed");
+  res.send("Hello World");
+});
+
+app.get("/todos", (req, res) => {
+  const showTodosFlag = await res.locals.bucketUser.getFlag("show-todos");
```
Comment (pavkam) [inline packages/node-sdk/example/app.ts:32]: Wait, did I add a separate `getFlag`? I though not. Also, we discussed the `global` flag type, I opted to make the `getFlags` generic. Should we exemplify that by declaring a `type AppFlags = 'show-todos' | 'something'` and

Should we make the `BucketClient` generic or just the flag getting function?

File: packages/node-sdk/example/bucket.ts (hunk 1 above)
Comment (roncohen) [reply packages/node-sdk/example/bucket.ts:7]: with modern Node.js it should work.

File: packages/node-sdk/example/bucket.ts (hunk 2 above)
Comment (roncohen) [reply packages/node-sdk/example/bucket.ts:4]: yes, good idea

**Comment by roncohen** [comment]
As usual, fighting the "build system" to get everything compiled etc.

**Pull request merged**